### PR TITLE
[Dance Party Blockly] Instructions Markdown Center Block

### PR DESCRIPTION
**Pull request by epeach**
When trying Dance Party with Google Blockly, the blocks in instructions are rendered as translated by 16 pixels which cuts off some of the blocks. This change removes that translation to center the block within the svg and then adds a translation to the SVG to center it vertically.

Production with our Blockly Branch:
Screenshot from 2021-12-09 15-22-01

Production with Google Blockly:
Screenshot from 2021-12-09 15-22-19

After this change with Google Blockly:
Screenshot from 2021-12-09 15-23-01

After this change on our Blockly Branch:
Screenshot from 2021-12-09 15-26-32

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/blockly/googleBlocklyWrapper.js [hunk 1]
```diff
@@ -292,10 +292,24 @@ function initializeBlocklyWrapper(blocklyInstance) {
...
       // Shrink SVG to size of the block
       const bbox = svg.getBBox();
       svg.setAttribute('height', bbox.height + bbox.y);
       svg.setAttribute('width', bbox.width + bbox.x);
+      // Add a transform to center read-only blocks on their line
+      const notchHeight = workspace.getRenderer().getConstants().NOTCH_HEIGHT;
+
+      svg.setAttribute(
+        'style',
+        `transform: translate(0px, ${notchHeight + 7}px)`
```
Comment (epeach) [inline apps/src/blockly/googleBlocklyWrapper.js]: The block has a height of 32 pixels and the line height of the text is 25, which is how I get this 7, but I'm not sure if there's a better way to represent this that isn't a 'magical number'.

Comment (ajpal) [reply apps/src/blockly/googleBlocklyWrapper.js]: can you declare the 7 as a `const` variable doing something like `const blockPadding = lineHeight - textHeight` to make it more clear in the code where it came from?

Comment (epeach) [reply apps/src/blockly/googleBlocklyWrapper.js]: I couldn't find a way to do this through calculation, but I broke it out into a const with a comment, so hopefully that makes sense.

**Comment by epeach** [comment]
@ajpal - PTAL, when you get a chance! Thanks!

**Review by ajpal — APPROVED** [review]

**Pull request merged**